app/domain/config-types.ts

```diff
@@ -119,14 +132,27 @@ export type BarConfig = t.TypeOf<typeof BarConfig>;
 
 const ColumnFields = t.intersection([
   t.type({
-    x: GenericField,
+    x: t.type({
+      componentIri: t.string,
+      sorting: t.type({
+        sortingField: t.union([t.literal("alphabetical"), t.literal("y")]),
```

Comment:
I was confused by `alphabetical` here (because it implies order). Shouldn't the _field_ be named `label` or `categoryLabel` instead?